### ISSUE TITLE
Can you share training data?

### ISSUE THREAD

**Issue by zengbw** [xianyuMeng/FacialCapture#1]
Thanks for your code. But I would like to train the network by myself. Could you share the training data?

**Comment by xianyuMeng** [comment]
I'm afraid that I can not provide the traning data since the data belongs to Owlii (owlii.com). 
You can contact directly to the authors of the paper to inquire about it. Good luck.

**Comment by KangolHsu** [comment]
Can you show some example of the train data and ground truth label？
